examples/clojure/Earthfile

```diff
@@ -0,0 +1,18 @@
+VERSION 0.7 
+FROM clojure:temurin-8-lein
+WORKDIR /var/app
+
+build:
+    COPY earthly-example/ .
+    RUN lein uberjar
+    SAVE ARTIFACT target/uberjar/earthly-example-*-standalone.jar earthly-example-standalone.jar
+
+docker:
+    FROM openjdk:22-slim-bullseye
+    RUN apt update && apt install zip -y 
+    COPY +build/earthly-example-standalone.jar earthly-example-standalone.jar
+    EXPOSE 8080
+    ENTRYPOINT java -Dserver.port=8080 -jar earthly-example-standalone.jar
+    COPY extractJarVersion extractJarVersion
```

Comment:
nit: can you move this script (and dependencies, e.g. `zip`) into the `build` target?

That way you can produce a smaller docker image (which won't include these helper scripts)

It could look something like:

```
build:
    RUN apt update && apt install zip -y 
    COPY earthly-example/ .
    RUN lein uberjar
    RUN ./extractJarVersion target/uberjar/earthly-example-*-standalone.jar > version
    SAVE ARTIFACT target/uberjar/earthly-example-*-standalone.jar earthly-example-standalone.jar
    SAVE ARTIFACT version

docker:
    FROM openjdk:22-slim-bullseye
    COPY +build/* .
    EXPOSE 8080
    ENTRYPOINT java -Dserver.port=8080 -jar earthly-example-standalone.jar
    SAVE IMAGE clojure-example:$(cat version)
```